types/angular-hotkeys/README.md

```diff
@@ -37,15 +37,15 @@ After installing a package you can import the definition file and begin using it
 
 ```ts
 
-import * as hotkeys from '../../node_modules/@types/angular-hotkeys'
+import { HotkeysProvider } from 'angular-hotkeys';
```

Comment:
We won't actually publish a custom README, maybe this content should be moved into `index.d.ts`?